manifest.webapp

```diff
@@ -1,7 +1,7 @@
 {
   "name": "j2me.js",
   "description": "j2me interpreter for firefox os",
-  "launch_path": "/index.html?autosize=1&logConsole=web",
+  "launch_path": "/index.html",
```

Comment:
> It looks like we don't default to autosize=1 anymore; does this cause apps to appear incorrectly on device and/or in emulators?

Yes, that's the functionality regression to which I was referring when I noted (obliquely), "It isn't entirely functionally equivalent, since the query parameters allow us to specify parameters that only apply when the app is installed as an app (as opposed to being loaded in the browser)."